.github/workflows/build-ci.yml

```diff
@@ -0,0 +1,33 @@
+name: Java SDK CI
+
+on: [push,pull_request]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    env:
+      CC_TEST_REPORTER_ID: 7603e87a55419aadb1e918a8c959fac1b43d45ef92be67a8e595015f2ff92882
+      COVERALLS_REPO_TOKEN: ${{ secrets.COVERALLS_REPO_TOKEN }}
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up openjdk
+        uses: actions/setup-java@v1
+        with:
+          java-version: 1.8
+      - name: register repo token
+        run: |
+          echo repo_token: ${COVERALLS_REPO_TOKEN} > ./.coveralls.yml
+      - name: Before script
+        run: |
+          curl -L https://codeclimate.com/downloads/test-reporter/test-reporter-latest-linux-amd64 > ./cc-test-reporter
+          chmod +x ./cc-test-reporter
+          ./cc-test-reporter before-build
+
+      - name: Build, Test and upload report to Coveralls
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: |
+          ./gradlew clean build jacocoTestReport
+          ./gradlew coveralls
+      - name: Upload to CoPilot
+        run: bash <(curl -s https://copilot.blackducksoftware.com/ci/githubactions/scripts/upload)
```

Comment:
this is missing the post steps as well


```
  - bash <(curl -s https://copilot.blackducksoftware.com/ci/travis/scripts/upload)
  - ./cc-test-reporter after-build --exit-code $TRAVIS_TEST_RESULT

```

Reply:
Added, for java code climate is asking to use different command since this has issue with it
https://docs.codeclimate.com/docs/test-coverage-troubleshooting-tips